src/libstd/net/tcp.rs

```diff
@@ -282,6 +284,20 @@ impl TcpStream {
     ///                        .expect("Couldn't connect to the server...");
     /// stream.set_read_timeout(None).expect("set_read_timeout call failed");
     /// ```
+    ///
+    /// An [`Err`] is returned if the zero [`Duration`] is passed to this
```

Comment:
I don't know if we need to have an explicit example of an invalid call.

Reply:
We talked about it at the docs team meeting, and everyone was generally in favor of showing off error conditions.